test/services/remove.director.error.keys.service.unit.ts

```diff
@@ -0,0 +1,46 @@
+import { mockValidationStatusResponse, mockValidationStatusResponseList, mockValidationStatusResponseList2, mockValidationStatusResponseList3, mockValidationStatusResponseList4 } from "../mocks/validation.status.response.mock";
+import { retrieveErrorMessageToDisplay, retrieveStopPageTypeToDisplay } from "../../src/services/remove.directors.error.keys.service";
+import { STOP_TYPE } from "../../src/utils/constants";
+
+describe("Test remove director error keys service", () => {
+
+  describe("retrieveErrorMessageToDisplay tests", () => {
+
+    it("Should return first web error message that matches priority order", async () => {
+      const newMessage = retrieveErrorMessageToDisplay(mockValidationStatusResponseList);
+      expect(newMessage).toEqual("Date director was removed must be a real date");
+    });
+  
+    it("Should return next web error message that matches priority order", async () => {
+      const newMessage = retrieveErrorMessageToDisplay(mockValidationStatusResponseList2);
+      expect(newMessage).toEqual("Enter a date that is in the past");
+    });
+  
+    it("Should return next web error message that matches priority order", async () => {
+      const newMessage = retrieveErrorMessageToDisplay(mockValidationStatusResponseList3);
```

Comment:
it.each(
    [
        ['Should return first web error message that matches priority order', mockValidationStatusResponseList, 
        "Date director was removed must be a real date"], 
        ['Should return next web error message that matches priority order', mockValidationStatusResponseList2, 
        "Enter a date that is in the past"]
    ])("%s", (title, mockValidationStatusList, expectedValue) => 
    { 
        const newMessage =  retrieveErrorMessageToDisplay(mockValidationStatusList); 
        expect(newMessage).toEqual(expectedValue);
    });